pom.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    
+    <groupId>com.lightstep.tracer</groupId>
+    <artifactId>java-common</artifactId>
+    <version>0.12.6</version>
+    <packaging>jar</packaging>
+    <description>The LightStep OpenTracing Tracer implementation for Java</description>
+    <url>https://github.com/lightstep/lightstep-tracer-java-common</url>
+
+    <organization>
+        <name>LightStep</name>
+        <url>http://lightstep.com/</url>
+    </organization>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://opensource.org/licenses/MIT</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <id>lightstep</id>
+            <name>LightStep</name>
+            <email>hello@lightstep.com</email>
+        </developer>
+    </developers>
+
+    <scm>
+        <connection>https://github.com/lightstep/lightstep-tracer-java-common</connection>
+        <developerConnection>https://github.com/lightstep/lightstep-tracer-java-common</developerConnection>
+        <url>https://github.com/lightstep/lightstep-tracer-java-common</url>
+    </scm>
+
+    <properties>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
```

Comment:
Apparently it supports 7, with a touch of 8 if you want. https://developer.android.com/guide/platform/j8-jack.html